src/main/kotlin/RealmModelGen.kt

```diff
@@ -57,6 +55,15 @@ class Model(val schema: Schema, val name: String) {
 
     var primaryKey: Property? = null
 
+    private val indexedProperties: List<Property>
+        get() = properties.filter { it.indexed }
+
+    val hasIndexedProperties: Boolean
+        get() = indexedProperties.isNotEmpty()
+
+    val indexedPropertiesString: String
+        get() = indexedProperties.joinToString(separator = ", ", transform = { "\"" + it.name + "\"" })
```

Comment:
```kotlin
 "\"" + it.name + "\""
```

 can change to

 ```kotlin 
"\"${it.name}\""`

Reply:
Thank you for the suggestion, addressed in 06bf194.